transcriptic/config.py

```diff
@@ -218,12 +218,13 @@ def api_root(self, value):
     @property
     def organization_id(self):
         try:
-            return self.env_args["org_id"]
+            return self.session.headers["X-Organization-Id"]
```

Comment:
Yup, these are equivalent since we set them both. It'll be nice if we could add an additional test that verifies that `env_args` is also set that we're no longer explicitly referring to it as the source of truth.

`env_args` is eventually used for resolving the routes, so we want to make sure that that behavior is retained.

Reply:
Added assertions for env vars with session headers value